OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyLogic.cs

```diff
@@ -826,4 +832,12 @@ class DropDownOption
 		public Func<bool> IsSelected = () => false;
 		public Action OnClick;
 	}
+
+	class DiscordDetails
+	{
+		public string Server { get; set; }
+		public string Map { get; set; }
+
+		public string GetDetails => $"{Map} - {Server}";
```

Comment:
IMO consistency outweighs brevity, so we should stick with the long form getter and `.F()` syntax if that is what is used in the rest of the file (unless there is a compelling reason to change it immediately). We should then tackle the syntax upgrades in a more systematic way, i.e. specific PRs to update entire files at a time, and require any new files to fully use the new syntax.